tests/FdServerTest.php

```diff
@@ -2,10 +2,11 @@
 
 namespace React\Tests\Socket;
 
-use Clue\React\Block;
 use React\Promise\Promise;
 use React\Socket\ConnectionInterface;
 use React\Socket\FdServer;
+// use function React\Async\await;
+// use function React\Promise\Timer\timeout;
```

Comment:
All looks good, but why this?

Reply:
Would have loved to use function imports for this update (requires PHP 5.6+ syntax), so left this in as a reminder for future self once we update minimum PHP version. Definitely looking forward to that, but you're right this should probably be left out from this PR in the meantime. Now removed to keep this simple.